README.md

```diff
@@ -52,7 +53,7 @@ To install this expansion, please follow these steps:
 In July 2019 we published a map expansion with 11 semantic layers (crosswalk, sidewalk, traffic lights, stop lines, lanes, etc.).
 To install this expansion, please follow these steps:
 - Download the expansion from the [Download page](https://www.nuscenes.org/download),
-- Move the four .json files to your nuScenes maps folder (e.g. `/data/sets/nuscenes/maps`).
+- Move the .json files to your nuScenes maps folder (e.g. `/data/sets/nuscenes/maps`).
```

Comment:
Actually, a lot of our code assumes that the maps are stored in a directory called `maps` under the dataroot directory. This makes it seem like the user has the option of storing the maps wherever they want.

Reply:
Just calling it nuScenes `maps` folder now.